src/feed-text-sanitization.c

```diff
@@ -134,6 +157,8 @@ eos_discovery_feed_sanitize_synopsis (const gchar *synopsis)
                                                CHARACTER_COUNT_THRESHOLD);
   g_autofree gchar *normalized_whitespace = normalize_whitespace (first_two_sentences);
   g_autofree gchar *with_ending_period = add_ending_period (normalized_whitespace);
+  g_autofree gchar *ellipsized = maybe_ellipsize_at (with_ending_period,
+                                                     CHARACTER_COUNT_ELLIPSIZE);
```

Comment:
I am uncomfortable hardcoding an ellipsize limit here, especially since the whole point of this library is to expose this data model to a wide variety of applications an platforms.
It should be the job of the UI layer to decide when/if to ellipsize...

Reply:
Makes sense. Right now `TextSanitization` is kind of like a quasi-UI layer. I think you make a good point here about trying to support different view sizes and locales though.

What we could do is allow the caller to specify the number of characters before new sentences are not "admitted" and then handle ellipsization entirely at the text layouting level in the UI ... do you know if Gtk has a way to ellipsize on word boundaries and not character boundaries at least?